Problem
Result
Check the error
Send Feedback
What error will be generated by the given code?
#include <iostream>
using namespace std;
void func(int a) {
    int b = a;
    b = b + 10;
}

int main() {
    int a = 10;
    func(a);
    cout << b << endl;
}
 info
You have max 2 attempts to score in this question.
Attempts left:
1/2
Options
This problem has only one correct answer
a is not declared in the scope.
b is not declared in the scope.
a and b both are not declared in this scope.
None of the above.
check_circle
Hurray! Correct Answer
Solution Description
The scope of b is limited to the function only but in the question we are trying to access it inside the main function, that's why error will be generated.

SUBMIT 
